backend/models.py

```
@@ -44,9 +46,7 @@ def place_mines(self):
             self.compute_neighbors(r, c)
 
     def compute_neighbors(self, r: int, c:int) -> None:
-        offsets = [(1, 0), (-1, 0), (0, 1), (0, -1), (1, 1), (1, -1), (-1, -1), (-1, 1)]
-
-        for r_offset, c_offset in offsets:
+        for r_offset, c_offset in self.offsets:
             new_r, new_c = r + r_offset, c + c_offset
 
             if self.is_inbound(r, c):
```

don't just return None, should return an Error class that has more specific information